app/lib/datetime.js

```diff
@@ -13,52 +13,32 @@ export default class DateTime {
   }
 
   _parse(date, format) {
-    // If `date` is already a Date object, there's nothing we need
-    // to do.
-    if (date instanceof Date) {
-      return date
-    } else if (typeof date === 'number') {
-      // If it's a number, we assume it's a timestamp, so we create a date
-      // from that. We'll assume the timestamp is in seconds, so we need to
-      // convert to milliseconds when passing to Date().
-      return new Date(date)
-    } else if (typeof date === 'string') {
-      // If there is a `format` specified, we'll try to parse the date using
-      // that.
-      if (format && format !== 'unix') {
-        try {
-          const parsedDate = fecha.parse(date, format)
-
-          // If the date was not parsed correctly, let's pass it through the
-          // parsing function again, but this time without a format. It may
-          // be that it's picked up as either a numeric timestamp or ISO8601.
-          if (parsedDate === null) {
-            return this._parse(date, null)
-          }
-
-          // When parsing a date from a string, it'll be parsed to the local
-          // time, which we do not want. So we get a new date that is computed
-          // by applying the timezone offset, which effectively means parsing
-          // the string as a UTC date.
-          return this._getLocalDateFromUTCDate(parsedDate)
-        } catch (err) {
-          console.error('Error parsing date', err)
-
-          return null
-        }
-      } else {
-        const intDate = parseInt(date)
+    if (date instanceof Date) return date
 
-        // It might be that the input date is still a timestamp, but represented
-        // as a string. If that's the case, we use the parsed int and treat it
-        // as a timestamp.
-        if (intDate.toString() === date) {
-          return new Date(intDate)
-        }
+    // Unix timestamp.
+    if (typeof date === 'number') return new Date(date)
 
-        // We assume the input string is in ISO8601 format.
-        return this._parse(date, FORMAT_ISO8601)
-      }
+    const intDate = parseInt(date, 10)
+
+    // Unix timestamp in a string.
+    if (intDate.toString() === date) return new Date(intDate)
+
+    try {
+      const parsedDate =
+        // Fecha requires a format. If no format is specified, try ISO.
+        fecha.parse(date, format || FORMAT_ISO8601) ||
+        // If format is specified but parsing fails, the value has probably been
```

Comment:
```suggestion

        // If format is specified but parsing fails, the value has probably been
```

Reply:
Nope, sorry, prettier doesn't like that :grin: